sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -181,6 +181,21 @@ class SQLConfSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("reset - single configuration") {
```

Comment:
Although this is an improvement PR, could you add a test prefix `SPARK-32406: ` here?